lib/ModelMixins/SearchProviders/LocationSearchProviderMixin.ts

```diff
@@ -32,8 +32,18 @@ function LocationSearchProviderMixin<
     @action
     showWarning() {}
 
-    supportsAutocomplete(): boolean {
-      return true;
+    async search(searchText: string, manuallyTriggered?: boolean) {
```

Comment:
Doubt: Could you explain when `manuallyTriggered` is true and why we need to differentiate both?

Reply:
`manuallyTriggered` is when the user presses enter to start the search; in that case, we need to cancel the debounce and run the search without waiting